src/main/java/com/tstepnik/planner/repository/TaskRepository.java

```diff
@@ -14,4 +14,6 @@
     List<Task> findAllByUserId(Long userId);
 
     Optional<Task> findById(Long id);
+
+    Task getById(Long id);
```

Comment:
I believe you don't need to add this method on your own since it might be inherited from JpaRepository

Reply:
Actually I think the method JpaRepo has is called `getOne`

Reply:
Yes, you're right. I changed it.